rllib/core/learner/learner.py

```diff
@@ -1471,14 +1473,17 @@ def save_state(self, path: Union[str, pathlib.Path]) -> None:
 
         the state of the learner is saved in the following format:
 
-        checkpoint_dir/
-            learner_state.json
-            module_state/
-                module_1/
-                    ...
-            optimizer_state/
-                optimizers_module_1/
-                    ...
+        .. testcode::
+            :skipif: True
```

Comment:
You can just do `code-block::`. Ratched will only complain for `code-block:: python`